homework-solutions/js/hw-5/task-1.js

```diff
@@ -6,6 +6,17 @@
 
 let evenNumbersResult = '';
 
+for (let i = 10; i >= 0; i--) {
+    if (i % 2 !== 0) {
+        continue;
+    }
+    evenNumbersResult += i;
+    if (i !== 0) {
+        evenNumbersResult += '-';
+    }
+}
```

Comment:
Как вариант, убираем лишнее использование `continue` 
```suggestion
for (let i = 10; i >= 0; i--) {
  if (i % 2 === 0) {
    if (evenNumbersResult) {
      evenNumbersResult += '-';
    }
    evenNumbersResult += i;
  }
}
```